server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -44,6 +44,19 @@ export function validateTokenClaims(
     return claims;
 }
 
+/**
+ * Generates a document creation JWT token, this token doesn't provide any sort of authorization to the user.
+ * But it can be used by other services to validate the document creator identity upon creating a document.
+ */
+export function getCreationToken(token: string, key: string, documentId: string, lifetime = 5) {
```

Comment:
From the code it looks like `lifetime` is in [seconds](https://github.com/microsoft/FluidFramework/blob/d8f825ec284baf904912a2cbb82c87164ed463e8/packages/drivers/local-driver/src/auth.ts#L29). Nevertheless, It should be `5 * 60`, but I made a mistake when I changed this file to reuse `generateToken`.  
The idea is not to have long lifetime for creation token compared to the normal tokens, because ideally it should be used only once and immediately after document creation.

Reply:
Thanks for checking me on that. You're totally right that it is in seconds